0x/Utilities/EthereumAddress.cs

```diff
@@ -7,13 +7,17 @@ namespace ZeroX.Utilities
 {
     public class EthereumAddress
     {
+
+        // TODO: Add docs
+
         private string _address;
         private const string ZeroAddressLiteral = "0x0000000000000000000000000000000000000000";
```

Comment:
Also in a constants file (reuse if you have it in some dependency).

Reply:
Same as above